doc/languages-frameworks/beam.section.md

```diff
@@ -79,6 +212,72 @@ mkShell {
 }
 ```
 
-#### Building in a Shell (for Mix Projects) {#building-in-a-shell}
+#### Elixir - Phoenix project
+
+Here is an example `shell.nix`.
+
+```shell.nix
+# latest elixir version are only available on unstable
+with import <nixpkgs-unstable> { };
```

Comment:
Same concern as above.

Reply:
I guess it makes sense to put it to just nixpkgs since by the time the doc is released, this should be available on stable.